### PR TITLE
mono 4.8.0.495, fsharp 4.1.0.0

### PR DESCRIPTION
**Pull request by mfilippov**
mono 4.8.0.495, fsharp 4.1.0.0

### CODE REVIEW

File: Formula/mono.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
 class Mono < Formula
   desc "Cross platform, open source .NET development framework"
   homepage "http://www.mono-project.com/"
-  url "https://download.mono-project.com/sources/mono/mono-4.6.2.16.tar.bz2"
-  sha256 "8965d107f4ebf4583ba1b50e0dcad39f0dc6adac8df7a083e9c5879ad93c0ea4"
+  url "https://download.mono-project.com/sources/mono/mono-mono-4.8.0.495.tar.bz2"
+  sha256 "7ba62e6f42559d58dc447a19fc1cb2a9c7977d6c6e21e1e335f73917dea120cb"
```
Comment (neutric) [inline Formula/mono.rb]: Typo, should be `url "https://download.mono-project.com/sources/mono/mono-4.8.0.495.tar.bz2"`. Hash is correct.

File: Formula/mono.rb [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
   bottle do
-    sha256 "c7b2d897cf3d10a758eaeb59bf82b482b63ba322885b656668b75fc44c59c719" => :sierra
-    sha256 "3dbee03cc6968243075f7833aa8af5148a321f9e8e48c9fa7746924d18a57ff3" => :el_capitan
-    sha256 "4e85f1f848084cd667490903f90dbf8b2c0b085f4554031ba41e2354234927dc" => :yosemite
+    sha256 "dd214b33b7b66785327aecaac80d2260cca4e5e97dbd0efcb0d2c04d9eba18a9" => :sierra
+    sha256 "c57fa4bae502fadb7022f7f00517e5519d0fc940a6ae27ab576651d29cf201dd" => :el_capitan
+    sha256 "06bddaf75d257fcc6c2faa05b161790cabd79ef0e7e55651632a87c64e050ec9" => :yosemite
```
Comment (neutric) [inline Formula/mono.rb]: Leave those bottle hashes untouched, they are done for you automatically.

**Comment by mfilippov** [comment]
Fixed

**Comment by nijikon** [comment]
@BrewTestBot test this please

**Comment by nijikon** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
aborted after almost 2 hours

**Comment by mfilippov** [comment]
@BrewTestBot test this please

**Comment by mfilippov** [comment]
I verify build local with brew test-bit it is ok.

**Comment by mfilippov** [comment]
@nijikon @ilovezfs  I do not understand why the build failed. In console output, I see only: "sierra is still in the queue: Waiting for next available executor on sierra"

**Comment by nijikon** [comment]
This actually fails for me locally.

**Comment by mfilippov** [comment]
@nijikon Interesting, could you show me an error?

**Comment by fxcoudert** [comment]
Building fine here locally on Sierra: https://gist.github.com/fxcoudert/18087fea813db4a928ade11da76436b0

@BrewTestBot test this please

**Comment by ilovezfs** [comment]
To be clear, the problem is that the tests hang.

**Comment by ilovezfs** [comment]
Testing locally the test hangs as well.
```
brew test --verbose mono
```

If you run the command manually, it just appears to be entering interactive mode.
```
joe$ bash
bash-4.4$ printf '"Hello Homebrew"; exit 0' | fsharpi

F# Interactive for F# 4.1
Freely distributed under the Apache 2.0 Open Source License

For help type #help;;

> - 
```

**Comment by mfilippov** [comment]
@ilovezfs Thanks I try to fix it.

**Comment by mfilippov** [comment]
I found problem try to fix.

**Comment by mfilippov** [comment]
Test passed locally

**Comment by mfilippov** [comment]
I think a previous build should be stopped.

**Comment by mfilippov** [comment]
fsi behaviour changed in 4.1 it is problem for fsharpi test

**Comment by JCount** [comment]
I stopped the build @mfilippov

**Comment by mfilippov** [comment]
Fixed

**Comment by fxcoudert** [comment]
Thanks @mfilippov for your contribution, and to all who helped!

**Pull request closed without merging**